ansible/configs/three-tier-app/sample_vars/private-dns-osp.yml

```diff
@@ -0,0 +1,161 @@
+# sample vars configuration file to deploy three-tier-app to OSP
+#
+# Only the bastion is exposed and the other hosts do not have public DNS or
+# Floating IPs
+# Consider exposing Frontends via public_dns: true in the instance dictionary
+#
+# Usage: ansible-playbook main.yml -e @configs/three-tier-app/sample_vars/private-dns-osp.yml
+# 
+# Ideally make and keep a copy OUTSIDE your repo, especially if using Cloud Credentials
+# Credentials can also be set seperately i.e. ~/secrets.yml and passed in with
+# a 2nd `-e` argument i.e. -e ~/secrets.yml
+# You may also need to pass other secret vars like `own_repo_path`
+
+env_type: three-tier-app        # Name of config to deploy
+output_dir: /tmp/output_dir     # Writable working scratch directory
+email: name@example.com         # User info for notifications
+
+guid: tok01                     # Your Global UNIQUE Identifier, change
+
+repo_method: file
+own_repo_path:                  # Move this var and set into your secrets.yml
+
+# Cloud specfic settings - example given here for OSP
+# ec2 example typically in ../sample_vars.yml
+
+cloud_provider: osp             # Which AgnosticD Cloud Provider to use
+ansible_user: cloud-user        # Default user in an OSP environment
+remote_user: cloud-user         # Default user in an OSP environment
+
+# The domain that you want to add DNS entries to
+osp_cluster_dns_zone: blue.osp.opentlc.com
+
+# The dynamic DNS server you will add entries to.
+# NOTE: This is only applicable when {{ use_dynamic_dns}} is true
+osp_cluster_dns_server: ddns01.opentlc.com
+
+# Instance type - you can list these with `openstack image list`
+
+bastion_instance_type: 2c2g30d
+app_instance_type: 2c2g30d
+appdb_instance_type: 2c2g30d
+frontend_instance_type: 2c2g30d
+support_instance_type: 2c2g30d
+
+#___image: rhel-guest-7.7u2    # blue
+___image: rhel-server-7.7-update-2  # red
```

Comment:
Just a suggestion...make this var with less _. I don't get the reason for such a long prefix before image since it would be easy to miss one and cause failures when other vars rely on this.

Reply:
that's me.

The purpose of this var was just not to repeat myself in the following lines. The scope of this var was just 3-4 lines and should not be used anywhere else, i just added quickly `_`s because 'image' was too generic and i could have risk of collision.

Feel free to rename to anything that makes sense, or just duplicate the string maybe and remove the var.